From 0de8463876415035cf0017ad1ca34c5ee82f6e1d Mon Sep 17 00:00:00 2001
From: Kyle Harrison <khwebmail@gmail.com>
Date: Sun, 15 Dec 2019 12:21:05 +0000
Subject: [PATCH] [DNM] Turn off secure ADB

---
 config/common.mk | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 6da2c44f..ac13264e 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -18,13 +18,13 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.config.notification_sound=Argon.ogg \
     ro.config.alarm_alert=Hassium.ogg
 
-ifeq ($(TARGET_BUILD_VARIANT),eng)
+#ifeq ($(TARGET_BUILD_VARIANT),eng)
 # Disable ADB authentication
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
-else
+#else
 # Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
-endif
+#PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
+#endif
 
 # Backup Tool
 PRODUCT_COPY_FILES += \
-- 
2.24.1

